### DISCUSSION TITLE
Zigbee Bubble Smasher!

### DISCUSSION THREAD

**Discussion by chunkysteveo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#539] [Show and tell]
![20230609_215025](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/14663002/35cd6a0f-c792-44e1-ae18-1c254ee518cb)

Bubble Smasher! 

Zigbee door sensor to Zigbe2mqtt to Home Assistant to the Hot Tub! Smash to turn bubbles on... twist to end the fun!

**Comment by mrjeth** [comment]
Very interested in how you did it and interacted with MQTT. Pllease share your HA Automations and Yaml.
Brg
Tomas

**Comment by chunkysteveo** [comment]
There's no MQTT configuration or YAML needed, just a couple of automations using Home Assistant to trigger on and off. (Assuming you have Zigbee2MQTT running well?)

Currently have two automations (which I should tidy up to one using trigger IDs....Lazy!!). 

Automation ON trigger is my "door sensor" contact opening (i.e. I hit the button which opens the reed switch) and the action is to turn on bubbles on Layzspa (If it is connected and bubbles are already not on).

Automation OFF trigger is my "door sensor" contact closing (i.e. I twist the button off which closes the reed switch) and the action is to turn off bubbles on Layzspa (If it is connected and bubbles are already on).

Here's the ON Automation (My Door Sensor is changed to a "Plug" via Home Assistant GUI, so reports "unplugged" etc, not contact open closed).
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/14663002/72373da7-02a1-420a-8690-f1c6a1102e65)

Let me know if you want any more info or pointers. I need to tidy up the automations into one using trigger IDs and multiple actions - but I am lazy! haha

**Reply by mrjeth** [reply]
Thanks for sharing!
I have an idea where it would be nice if it was possible to set the desired temperature. Do you know how to do it from HA?
Yes I have installed MQTT and it seem to work nicely!

**Reply by chunkysteveo** [reply]
> Thanks for sharing! I have an idea where it would be nice if it was possible to set the desired temperature. Do you know how to do it from HA? Yes I have installed MQTT and it seem to work nicely!

The temperature should be setable via the climate thermostat that is added to HA using the new firmware...?

**Reply by mrjeth** [reply]
Thank you for directing me in a good direction. It works!

I also figured out how to start/stop the pump in an automation .
Brg
Tomas

**Reply by chunkysteveo** [reply]
NO worries, you're welcome! I need to set up a pump automation to stop it running 24x7 - some some pennies.... 🤑

**Comment by mrjeth** [comment]
It is not for the great saving it is the night sleep 🤭

Den tis 13 juni 2023 10:00Stephen Ludgate ***@***.***> skrev:
